text_handler.py

```diff
@@ -86,18 +88,45 @@ async def __handleMusicGrabber(self, message: types.Message):
                 await message.delete()
         except music_grabber.WrongFileSizeError as e:
             logger.error(str(e))
-            inform = await message.reply("This won't be downloaded because "
-                                         "its audio file size is greater "
-                                         "than 50M",
-                                         disable_notification=True)
-            await self.__deleteMessage(60, inform)
+            text = "This won't be downloaded because its audio file size " + \
+                "is greater than 50M"
+            inform = await message.reply(text, disable_notification=True)
+            await self.__delete_messages(60, (inform, ))
         except music_grabber.WrongCategoryError as e:
             logger.error(str(e))
         except Exception as e:
-            await message.reply("<b>Error</b>: <code>" + str(e) +
-                                "</code>", parse_mode=ParseMode.HTML,
-                                disable_notification=True)
+            markup = self.__make_inline_keyboard()
+            error_html = "<b>URL</b>: " + message.text + \
+                "\n<b>Error</b>: <code>" + str(e) + "</code>"
+            await message.reply(error_html, parse_mode=ParseMode.HTML,
+                                disable_web_page_preview=True,
+                                disable_notification=True, reply_markup=markup)
 
-    async def __deleteMessage(self, delay, message):
+    def __make_inline_keyboard(self):
+        markup = types.InlineKeyboardMarkup(reasize_keyboard=True)
+        item1 = types.InlineKeyboardButton("Report Error",
+                                           callback_data=ERROR_REPORT)
+        item2 = types.InlineKeyboardButton("Delete Only",
+                                           callback_data=ERROR_DELETE)
+        markup.add(item1, item2)
+        return markup
+
+    async def callback_error_message(self, callback: types.CallbackQuery):
+        from_chat_id = callback.message.chat.id
+        message_id = callback.message.message_id
+        callback_data = callback.data
+        if callback_data == ERROR_REPORT:
+            answer_text = "Successfully forward the error message to dev!"
+            await self.bot.forward_message(chat_id=DEV_ID,
+                                           from_chat_id=from_chat_id,
+                                           message_id=message_id)
+            await self.bot.answer_callback_query(callback.id, text=answer_text,
+                                                 show_alert=False)
+            await self.__delete_messages(3, (callback.message, ))
```

Comment:
magic numbers is a bad practice, you should make a constant for this number **3** DELETE_DELAY_IN_SEC

then it'll look like 
`await self.__delete_messages(DELETE_DELAY_IN_SEC, (callback.message, ))`

Reply:
came up with a better name, see this comment https://github.com/konnovdev/cpop_bot/pull/13#discussion_r543539561

Reply:
constant? the value of constant can't be changed.

`__delete_messages` is used three times in text_handler.py
- "ping" and "pong" are deleted after 5 seconds (I added this in this PR but forgot to tell you)
- "This won't be downloaded because its audio file size %s is greater than 50M" is deleted after 60 seconds
- the error message will be delete 3 seconds after display "Message forwarded to the developer"

we can change the duration if you have better idea. from my test that 3 seconds seems fine, it feels a little bit uncomfortable to delete that message right after show the text "message forwarded to the developer".

using variable seems too much for just explain that. I think it's better to change the method name to `__delete_messages_after_seconds` or `delay_delete_messages`?

I don't really like it though, I think maybe it's better to name it `del_msg_sec` or `delay_del_msg` and add docstring to the function to explain ```"""Delete messages after specified seconds"""```